flink-runtime-web/web-dashboard/package.json

```diff
@@ -14,51 +14,50 @@
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "^14.2.6",
-    "@angular/common": "^14.2.6",
-    "@angular/compiler": "^14.2.6",
-    "@angular/core": "^14.2.6",
-    "@angular/forms": "^14.2.6",
-    "@angular/platform-browser": "^14.2.6",
-    "@angular/platform-browser-dynamic": "^14.2.6",
-    "@angular/router": "^14.2.6",
+    "@angular/animations": "^18.2.12",
+    "@angular/common": "^18.2.12",
+    "@angular/compiler": "^18.2.12",
+    "@angular/core": "^18.2.12",
+    "@angular/forms": "^18.2.12",
+    "@angular/platform-browser": "^18.2.12",
+    "@angular/platform-browser-dynamic": "^18.2.12",
+    "@angular/router": "^18.2.12",
```

Comment:
You can rebase the latest code, currently the front-end dependencies have been upgraded in this PR #25713

Reply:
@simplejason Thank you, rebased the PR to take these changes